packages/node/src/utils/substrate.test.ts

```diff
@@ -29,6 +31,29 @@ describe('substrate utils', () => {
     }
   });
 
+  it('filters blocks based on timestamp', async () => {
```

Comment:
Can we get a test with an invalid filter too please? I want to see what error handling is like